specification/08-attestation.adoc

```diff
@@ -1,2 +1,94 @@
 [[attestation]]
 == Device Attestation
+
+There are two attestation involved in the CoVE-IO architecture.
```

Comment:
How about we change to `two attestation stages`?

I have concern on using `model`, because `two models` may make people there are 2 possibilities and they can choose 1 of them.

Reply:
Makes sense to me, yes.

Reply:
done